src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
This is the source code meta data section used by my Web Low lever Language (WLL) project, necessarily being developed externally to this group because of seemingly irreconcilable difference and because it seem practical. Some limited discussion is at https://github.com/JSStats/wll and I am working to put together a more formal site, and perhaps another community group to make it a web standard too. It missed G's document, but is included in v8 and the groups binary encoding as a section to ignore. I'd like the binaryen tools to at least not abort on these source code files. The groups spec does specify that 'unknown' sections are to be ignored, and this might be abstracted to just another ignored section from the groups pov.